compatibility_server/pip_checker.py

```diff
@@ -27,29 +27,37 @@
 import enum
 import json
 import logging
-import os.path
 import shlex
-import subprocess
-import tempfile
 import urllib.request
 
 from typing import Any, List, Mapping, Optional
 
+import docker
+
 PYPI_URL = 'https://pypi.org/pypi/'
+CONTAINER_WITH_PKG = "checker"
 
 
-class PipError(Exception):
-    """A pip command failed in an unexpected way."""
+class PipCheckerError(Exception):
```

Comment:
Maybe you should have PipCheckerError which just takes an error string (for use in the docker failure case) and a subclass, PipError, which takes the command.

Reply:
Done.